rtf_vis_tool/README.md

```diff
@@ -0,0 +1,47 @@
+# GTSFM Pipeline Visualization Tool
+
+This project was created with React for viewing metrics related to the various processes within the GTSFM pipeline. Also used to view resulting point clouds using React Three Fiber.
```

Comment:
i might change the wording to "This viewer app was created..." since "the project" is GTSFM in general